build.xml

```diff
@@ -57,7 +93,15 @@
 			<condition>
 				<not>
 					<isset property="env.FLEX_HOME"/>
-			  </not>
+				</not>
+			</condition>
+		</fail>
+		<echo>UglifyJS will be invoked as ${uglifyjs}</echo>
+		<fail message="Could not find ${uglifyjs} in your PATH! Please get node.js and run `npm i -g uglify-js`">
```

Comment:
I wouldn't. It'd make the build unpredictable and significantly increase the build size. If UglifyJS isn't available locally, I would at most fall back to an online version or install it automatically.